lgc/elfLinker/RelocHandler.cpp

```diff
@@ -216,7 +216,7 @@ bool RelocHandler::getValue(StringRef name, uint64_t &value) {
     return true;
   }
   if (name == reloc::ShadowDescriptorTableEnabled) {
-    value = m_pipelineState->getOptions().shadowDescriptorTable != ShadowDescriptorTableDisable;
+    value = m_pipelineState->getOptions().enableFmask;
     return true;
   }
```

Comment:
Same here, this about where to find the fmask, not whether to use an fmask. Therefore, it shouldn't be changed.